### DISCUSSION TITLE
[solved] is it possible to set hyprland background to solid grid color without using any external applications or tool?

### DISCUSSION THREAD

**Discussion by pastquerier** [hyprwm/Hyprland#6109] [Q&A] [answered]
I want to do it the same way hyprland do it for their default wallpaper, if you know any other way please tell me.
 Thank you.
edit: solved it by adding "background_color" variable to "misc" section
exemple: 
misc {
    force_default_wallpaper = 0	
    disable_hyprland_logo = true
    background_color = 0x222222
}

**Comment by vaxerski** [comment, marked as answer]
https://wiki.hyprland.org/Configuring/Variables/#misc ctrl+f background

**Reply by pastquerier** [reply]
Thank you.
